src/sync/sync-api.js

```diff
@@ -50,7 +56,8 @@ export class SyncApi extends TypedEmitter {
   #pscByPeerId = new Map()
   /** @type {Set<string>} */
   #peerIds = new Set()
-  #isSyncing = false
+  #wantsToSyncData = false
+  #isBackgrounded = false
```

Comment:
I think this naming (`isBackgrounded`) is confusing, because it doesn't describe what this does, it describes a state that only really makes sense on mobile, and it's behaviour is determined by platform specifics of how we want sync to behave when in the background. Understanding what it does requires reading the code, or understanding that we want sync to gracefully stop when complete when the app is in the background. I'm thinking ahead to when we integrate this into Desktop and Mapeo Cloud server. This is why in my hacking around yesterday I was moving towards the concept of starting and stopping both sync types (initial & data), and being more explicit about auto-starting initial sync on initialization (e.g. starting initial sync in the constructor).

I'm not great at names, but perhaps we can call this either `wantsToStopAllSync` or the inverse `wantsToSyncInitial` or `wantsToSyncPresync`. Then perhaps the methods can be `kStopAllSyncGracefully` and `kResumeSync`?

Reply:
Renamed in 4a641e77dc74e9b316201f4c667fa5fcad2eb52e.